ui/app/AppLayouts/Profile/views/AboutView.qml

```diff
@@ -7,7 +7,6 @@ import StatusQ.Core.Theme 0.1
 import StatusQ.Controls 0.1
```

Comment:
Belongs to another PR

Reply:
will disapear after the other PR is merged and this one is rebased